torch/csrc/jit/codegen/cuda/codegen.cpp

```diff
@@ -754,10 +754,12 @@ class CudaKernelGenerator : private OptOutConstDispatch {
     auto out_tv = rop->output(0)->as<kir::TensorIndex>()->view();
     auto index = genTensorIndex(rop->getPhiloxIndex()->as<kir::TensorIndex>());
     int multiple = out_tv->getDataType() == DataType::Double ? 2 : 4;
-    indent() << "nvfuser_index_t rng_subseq" << rop->name() << " = (" << index
-             << ") / " << multiple << ";\n";
-    indent() << "nvfuser_index_t rng_component" << rop->name() << " = ("
-             << index << ") % " << multiple << ";\n";
+    indent() << "nvfuser_index_t linear_index" << rop->name() << " = " << index
+             << ";\n";
+    indent() << "nvfuser_index_t rng_subseq" << rop->name() << " = linear_index"
+             << rop->name() << " / " << multiple << ";\n";
+    indent() << "nvfuser_index_t rng_component" << rop->name()
+             << " = linear_index" << rop->name() << " % " << multiple << ";\n";
```

Comment:
There is no real change in this part. Just feeling that
```C++
nvfuser_index_t linear_index123 = ...;
nvfuser_index_t rng_subseq123 = linear_index123 / 4;
nvfuser_index_t rng_component123 = linear_index123 % 4;
```
is more readable than
```C++
nvfuser_index_t rng_subseq123 = (...) / 4;
nvfuser_index_t rng_component123 = (...) % 4;
```